HIP/hip-646.md

```diff
@@ -0,0 +1,90 @@
+---
+hip: 646
+title: Fungible Token Metadata Field
+author: May Chan (@rocketmay)
+working-group: Ashe Oro (@Ashe-Oro)
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2023-01-04
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/627
+updated: 2023-01-04
+---
+
+## Abstract
+
+This HIP proposes the addition of the metadata field to Fungible Tokens (FT), taking after the Non-Fungible Token (NFT) metadata field which was added in HIP-17.
+
+## Motivation
+
+Similar to NFT's on the network, the fungible token implementation is largely functional. There is a need for a token creator to specify additional "metadata" about the token. The data and formatting is explored in HIP-405 but there is no field suitable for storing the link to this metadata. 
+
+The most 'logical' field proposed to store this data is the existing 'memo' field, however a dedicated metadata field is more appropriate.
+
+## Rationale
+
+The success of HIP-412 in standardizing the NFT space on Hedera demonstrates the potential of standardized metadata functionality for FT's. Wallets, explorers and other applications can display relevant supplementary information about tokens to users in a decentralized fashion.
+
+## User stories
+
+As a token creator, I want a platform-standardized method to host information about my token in a decentralized fashion, allowing applications to query data such as token icon and website.
+
+As a wallet or explorer, I want to query a mirror node to obtain the metadata for any fungible token, similar to current non-fungible token calls, to simplify calls and implementation.
```

Comment:
Here we might also clarify that current mirror nodes return metadata for individual NFTs; but we want to also get token-level metadata (especially for fungible token types).

Reply:
+1